string query
---
string[] names
OntologeniusSparqlIndexResponse[] results
string error
